### ISSUE TITLE
Colors dont adhere to design system

### ISSUE THREAD

**Issue by sanjeev-** [clickclickonsal/myhomiesformyhomies#2]

**Comment by sanjeev-** [comment]
is this still being maintained?

**Comment by clickclickonsal** [comment]
Yes, we're still under construction. We'll be sure to update all of our homies when it's live. 😄

**Comment by clickclickonsal** [comment]
Thank you for the feedback, we'll forward this issue to the engineering team & will take it into consideration during our next iteration.

**Comment by sanjeev-** [comment]
when is launch date

**Comment by clickclickonsal** [comment]
> when is launch date

We have something in the works so real soon! You'll be the first to know!
